stdlib/3/asyncio/windows_utils.pyi

```diff
@@ -12,7 +17,7 @@ class PipeHandle:
 
     def __init__(self, handle: int) -> None: ...
     def __repr__(self) -> str: ...
-    def __del__(self) -> None: ...
+    def __del__(self, _warn: _WarnFunction = ...) -> None: ...
```

Comment:
The extra parameter exists in 3.8, but not in 3.7. While I did not check other versions, I think it's safe to assume >= 3.8.